rythm.js

```diff
@@ -77,7 +77,7 @@ var Analyser = function Analyser() {
     });
     var scale = max - min;
     var actualValue = _this.frequences[index] - min;
-    var percentage = actualValue / scale;
+    var percentage = scale === 0 ? 0 : actualValue / scale;
```

Comment:
You're right ! Althougth there is only a really small chance that it could happen, this is safer this way.

Reply:
the dance effect change too fast to find this issue